src/tests/posts.test.js

```diff
@@ -16,9 +16,25 @@ describe("posts tests", () => {
     );
   });
 
+  test("check if lang is set to default", () => {
+    expect(config.dev.lang).toBe("en");
+  });
+
+  test("if invalid json is passed, expect false", () => {
+    expect(setConfig("invalid_file")).toBeFalsy();
+  });
+
   test("if data is passed, expect false", () => {
     expect(isNullOrWhitespace("test data")).toBeFalsy();
   });
+
+  test("if null is passed, expect true", () => {
+    expect(isNullOrWhitespace(null)).toBeTruthy();
+  });
+
+  test("if whitespace is passed, expect true", () => {
+    expect(isNullOrWhitespace(" ")).toBeTruthy();
+  });
 });
 
 describe("snapshot testing", () => {
```

Comment:
Hi !

Is it supposed to be here ? I don't see any use for it. Could you remove it, please ?